atom/browser/native_window_views_win.cc

```diff
@@ -299,6 +300,59 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
   }
 }
 
+void NativeWindowViews::HandleSizingEvent(WPARAM w_param, LPARAM l_param) {
+  double aspect_ratio = GetAspectRatio();
+  if (fabs(aspect_ratio) < 0.0001) {
+    return;
+  }
+
+  RECT* window_rect;
+  window_rect = reinterpret_cast<RECT*>(l_param);
+  int width = window_rect->right - window_rect->left;
+  int height = window_rect->bottom - window_rect->top;
+
+  double result_width = 0;
+  double result_height = 0;
+
+  gfx::Size win_fit_size(width, height);
+  const gfx::Size min_fit_size = GetMinimumSize();
+  win_fit_size.SetToMax(min_fit_size);
+
+  switch ((UINT)w_param) {
+    case WMSZ_LEFT:
+    case WMSZ_RIGHT:
+      result_width = win_fit_size.width();
+      result_height = static_cast<double>(result_width) / aspect_ratio;
```

Comment:
Is this `static_cast<double>()` necessary? Doesn't `int / double` get promoted as `double / double` anyway?

Same question for next three static casts